sdk/communication/Azure.Communication.Rooms/src/Models/RoomParticipant.cs

```diff
@@ -40,7 +40,7 @@ internal RoomParticipant(RoomParticipantInternal roomParticipantInternal)
         /// <summary> Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. </summary>
         public CommunicationIdentifier CommunicationIdentifier { get; set; }
         /// <summary> Role Name. </summary>
-        public string Role { get; set; }
+        public Role? Role { get; set; }
```

Comment:
This should be RoleType. I see few more places we need to change on this file. can you refer to https://github.com/ACS-Rooms/azure-sdk-for-net/pull/6/files